data/sql/updates/pending_db_world/rev_1619681545150953400.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1619681545150953400');
+
+SET @STORMWIND_GUARD_SHAMAN_TRAINER_TEXT:='I don''t know what''s possessed you to look for a shaman... but I have seen someone who matches the profile. There''s a woman who calls herself a Farseer on the shore in the Valley of Heroes.';
+UPDATE `broadcast_text` SET `MaleText` = @STORMWIND_GUARD_SHAMAN_TRAINER_TEXT, `FemaleText` = @STORMWIND_GUARD_SHAMAN_TRAINER_TEXT WHERE `ID` = 18360;
```

Comment:
I would highly prefer if you do not edit broadcast_text as it is supposed to only hold sniffed text.

I would like it if you make the changes in creature_text (I think it is) and set Broadcast_text for that NPC to 0 in that table =)

Reply:
Whoops, sorry, I always forget about this 😂
Will edit later today 😀

Reply:
Funnily enough, the correct text was already in `npc_text`...
Anyway it's fixed now, thanks!